server/services/participants-helper.js

```diff
@@ -311,7 +311,7 @@ class ParticipantsFinder {
                 //  and statuses (unless when the status is 'unavailable', in this case
                 //  we handle in the upper OR array)
                 { or: [{ and: [userRegionQuery(this.user.regions, 'body.preferredLocation')] }] }),
-            'body.interested': 'yes',
+            // Removed existing interest filter => 'body.interested': 'yes',
```

Comment:
I'd prefer to just have this line removed and have the commit message/ticket/PR explain what happened

Reply:
I keep this as documentation because I don't know long term effect of this change, a full regression can ensure that.

Reply:
Thanks, missed that lots of similar doc comments